src/test/java/com/revature/app/dao/VisualizationDaoTest.java

```diff
@@ -1,5 +1,41 @@
 package com.revature.app.dao;
 
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.ArrayList;
+
+import javax.transaction.Transactional;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.annotation.Commit;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.web.WebAppConfiguration;
+
+import com.revature.app.model.Visualization;
+
+// @ExtendWith(SpringExtension)
+@WebAppConfiguration
+@DirtiesContext
+// @TestMethodOrder(MethodOrder.OrderAnnotation.class)
```

Comment:
These annotations can be replaced with the annotations below, since we're making use of Spring Boot, which often simplifies the process of testing. Refer to this as an example of information regarding this: [https://reflectoring.io/spring-boot-test/]

@ExtendWith(SpringExtension.class)
@SpringBootTest